src/code/SecretManagement.cs

```diff
@@ -125,15 +170,18 @@ protected override void EndProcessing()
                         this));
             }
 
-            if (!ShouldProcess(Name, VerbsLifecycle.Register))
+            if (!ShouldProcess(Name, "Register module as a SecretManagement extension vault for current user"))
             {
                 return;
             }
 
             // Resolve the module name path in calling context, if it is a path and not a name.
             var results = InvokeCommand.InvokeScript(
-                script: "param([string] $path) (Resolve-Path -Path $path -EA Silent).Path",
-                args: new object[] { ModuleName });
+                    script: @"param([string] $path) (Resolve-Path -Path $path -EA Silent).Path",
```

Comment:
Shouldn't ErrorAction be Ignore so that it doesn't show up in $error?

Reply:
Done